tests/testthat/test-tidyeval-across.R

```diff
@@ -58,6 +58,15 @@ test_that("across() translates functions", {
   )
 })
 
+test_that("across() captures anonymous functions", {
+  dt <- lazy_dt(data.frame(a = 1))
+
+  expect_equal(
+   capture_across(dt, across(a, function(x) log(x))),
+   quo_squash(list(a = call2(function(x) log(x), quote(a))))
```

Comment:
Could you construct the expected value a little more directly? I don't remember what `quo_squash()` does off the top of my head.

Reply:
Oops, it was a mistake to have it there, thanks for catching that. In this case it wasn't doing anything. I've removed it.